modules/apps/accessibility/accessibility-test/src/testFunctional/macros/AccessibilityMenu.macro

```diff
@@ -15,17 +15,71 @@ definition {
 				checkboxName = "Enable Accessibility Menu",
 				locator1 = "Checkbox#ANY_CHECKBOX");
 
-			if (IsElementPresent(locator1 = "Button#SAVE")) {
-				Click(locator1 = "Button#SAVE");
-			}
-			else {
-				Click(locator1 = "Button#UPDATE");
-			}
-
-			Alert.viewSuccessMessage();
+			SystemSettings.saveConfiguration();
 		}
 	}
 
+	macro checkOffState {
```

Comment:
AccessibilityMenu.checkOffState doesn't sound clear enough. I suggest renaming to something like assertCannotAccessAccessibilityMenuModal